docs/index.rst

```diff
@@ -1,16 +1,77 @@
+.. _baseband_docs:
+
 ********
 Baseband
 ********
 
-Welcome to the Baseband documentation! Baseband is a package for reading and 
+Welcome to the Baseband documentation! Baseband is a package for reading and
 writing VLBI and other radio baseband files, with the aim of simplifying and
-streamlining data conversion and standardization.
+streamlining data conversion and standardization.  It provides:
+
+- File input/output functions for supported radio baseband formats, listed under
+  :ref:`specific file formats <specific_file_formats_toc>`.
+- Data container classes for supported formats, which can be used for accessing
+  and manipulating data samples and for generating baseband formats from
+  user-defined data.  The latter functionality can be used for converting
+  between file formats.
+- Classes for reading from and writing to an ordered sequence of files as if
+  it was a single file.
+
+.. _using_baseband_toc:
+
+Basics
+======
+
+.. toctree::
+   :maxdepth: 1
+
+   install
+   tutorials/getting_started
+   tutorials/glossary
+   authors_for_sphinx
```

Comment:
I think this should go under a separate "Project Details", as with the Astropy docs.  The release changes contain links to the license and changelog, which should go under there too.